python/setup.py

```diff
@@ -163,6 +165,9 @@
 cmdclass.update(versioneer.get_cmdclass())
 cmdclass["build_ext"] = build_ext
 
+if not libcuml_path:
```

Comment:
should probably be relative to location of setup.py rather than cwd with `../`

Reply:
Note: This is the only missing todo for Sunday's commit

Reply:
Nevermind, had some free time and pushed it